lang/python38-devel/Portfile

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                python38-devel
+version             3.8.0rc1
+epoch               1
+revision            1
+categories          lang
+license PSF
```

Comment:
> @pmetzger @iEFdev there are still a few tiny details left in the `python38-devel` port.
> 
> More importantly, there are additional things that would have been useful to do at the same time as submitting the final `python38` port. For example, there aren't `py38-tkinter` and `py38-gdbm` subports yet, the livecheck should be changed so that it only picks up release versions.

@reneeotten Ok, I took the livecheck from py37 now, and added it. 

> Since he's maintaining all other Python ports I'm hoping that he is willing to adopt the `python38` port as well and would defer making other changes to him.

Yes, that'd be preferrable. I just saw it was released, and the port didn't have a maintainer, with other people doing the upgrades before me. Hade a few hours to spare at the time, so I started doing it. If it saves time/commits I can add that now already.

> If he does not want to, I can take a closer look at these things later.

So, how do you want to proceed? If you want/need, I can give you (both) access, if that saves you time/commits. Or just add this, and continue later on?